lib/iotaUtils.js

```diff
@@ -82,17 +82,7 @@ function manageConfiguration(apiKey, deviceId, device, objMessage, sendFunction,
     }
 
     function extractAttributes(results, callback) {
```

Comment:
After simplifying this function, it has ended basically in a pass-through to the callback function.

Maybe we can simplify even more, removeing extractAttributes() function?

Reply:
Fixed in https://github.com/telefonicaid/iotagent-json/pull/533/commits/fd29166d7c8e9df6b9a8ffad02b335ef63d1246f